packages/vuetify/src/components/VSheet/VSheet.ts

```diff
@@ -1,70 +1,99 @@
-// @ts-nocheck
-/* eslint-disable */
+// Utilities
+import { computed, defineComponent, h } from 'vue'
+import { convertToUnit } from '@/util/helpers'
+import makeProps from '@/util/makeProps'
+import propsFactory from '@/util/propsFactory'
 
-// Styles
-import './VSheet.sass'
+import {
+  makeBorderRadiusProps,
+  makeDimensionProps,
+  makeElevationProps,
+  makeTagProps,
+  useBorderRadius,
+  useDimension,
+  useElevation,
+  useTheme,
+} from '@/composables'
 
-// Mixins
-import BindsAttrs from '../../mixins/binds-attrs'
-import Colorable from '../../mixins/colorable'
-import Elevatable from '../../mixins/elevatable'
-import Measurable from '../../mixins/measurable'
-import Roundable from '../../mixins/roundable'
-import Themeable from '../../mixins/themeable'
+// Types
+export interface PositionProps {
+  bottom?: boolean | string
+  left?: boolean | string
+  position?: string
+  right?: boolean | string
+  top?: boolean | string
+}
 
-// Helpers
-import mixins from '../../util/mixins'
+// Composables
+const makePositionProps = propsFactory({
+  bottom: [Boolean, String],
+  left: [Boolean, String],
+  position: String,
```

Comment:
should there be validation here?

Reply:
we can, I wasn't sure the best way to do it.

Reply:
added